### PR TITLE
[Implement][CCurtis] Generics 2 Tasks

### PR DESCRIPTION
**Pull request by constantin73**
Added 2 Generic tasks. Please review

### CODE REVIEW

File: constantinCurtis/src/main/java/generics/first/task/Print.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package generics.first.task;
+
+public class Print {
+    public <T, S> void printArray(T[] t, S[] s) {
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/generics/first/task/Print.java]: static

Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/generics/first/task/Print.java]: please overload this method with single parameter (generic), use `Arrays.toString()`

File: constantinCurtis/src/main/java/generics/second/task/Book.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package generics.second.task;
+
+public class Book extends Media {
+    private int pages;
+    private String author;
+
+    public Book(String title, double price, int year, int pages, String author) {
+        super(title, price, year);
+        if (pages > 0)
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/generics/second/task/Book.java]: please add brackets for readability

File: constantinCurtis/src/main/java/generics/first/task/Main.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package generics.first.task;
+
+public class Main {
+    public static void main(String[] args) {
+        Print myPrinter = new Print();
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/generics/first/task/Main.java]: make Print methods static

File: constantinCurtis/src/main/java/generics/second/task/Main.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
+package generics.second.task;
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/generics/second/task/Main.java]: if you want to sort your tasks adjust paths please, could be `generics/task/(first/second)`

File: constantinCurtis/src/main/java/generics/second/task/Video.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+public class Video extends Media {
+    private String format;
+    private double duration;
+    private String directedBy;
+
+    public Video(String title, double price, int year, String format, double duration, String directedBy) {
+        super(title, price, year);
+        this.format = format;
+        if (duration > 0)
```
Comment (solonarieugeniu) [inline constantinCurtis/src/main/java/generics/second/task/Video.java]: brackets

File: constantinCurtis/src/main/java/generics/first/task/Print.java (hunk 1 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/generics/first/task/Print.java]: Thanks. resolved

File: constantinCurtis/src/main/java/generics/second/task/Book.java (hunk 2 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/generics/second/task/Book.java]: thank you, done

File: constantinCurtis/src/main/java/generics/first/task/Main.java (hunk 3 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/generics/first/task/Main.java]: thank you, done

File: constantinCurtis/src/main/java/generics/second/task/Video.java (hunk 5 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/generics/second/task/Video.java]: thank you, done

File: constantinCurtis/src/main/java/generics/first/task/Print.java (hunk 1 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/generics/first/task/Print.java]: Thanks for help, fixed

File: constantinCurtis/src/main/java/generics/second/task/Main.java (hunk 4 above)
Comment (constantin73) [reply constantinCurtis/src/main/java/generics/second/task/Main.java]: Changed, thanks!

**Review by IIachimov — APPROVED** [review]

**Review by gaenache — APPROVED** [review]

**Review by abadarau — APPROVED** [review]

**Review by asecu — APPROVED** [review]

**Pull request merged**